web/src/helpers/cadPackages/openScadController.js

```diff
@@ -1,34 +1,50 @@
+const openScadBaseURL =
+  'https://x2wvhihk56.execute-api.us-east-1.amazonaws.com/dev'
```

Comment:
wasn't sure if I should commit this.
I can make it an env var instead, but that makes it a pain for anyone new forking the repo, as they'ed need to get it off me before they can use openscad locally. This isn't a secret so I think it's fine.